### PR TITLE
Upgraded dbal to version 2.9 and required doctrine/annotations.

### PR DESCRIPTION
**Pull request by julien-sebire**
How to test:
```
git clone git@github.com:oat-sa/package-tao.git
cd package-tao
git checkout develop
```
In `composer.json` at line 45, replace `"oat-sa/generis" : "dev-develop"` with `"oat-sa/generis" : "dev-feature/NEX-121/upgrade-dbal"`
```
composer install
```
This should install `doctrine/dbal` v 2.9.x and `doctrine/annotations` v 1.7.x.
Perform a standard installation of Tao and run it in your browser. It should behave normally.

### CODE REVIEW

**Review by ekkinox — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -56,7 +56,8 @@
   "require": {
     "clearfw/clearfw": "~1.2.0",
     "easyrdf/easyrdf": "^0.9.0",
-    "doctrine/dbal": "~2.5.0",
+    "doctrine/dbal": "^2.9.0",
+    "doctrine/annotations": "1.*",
```
Comment (ekkinox) [inline composer.json]: see https://packagist.org/packages/doctrine/annotations#v1.6.1
it implicitly bind you code to php 7.1 as a dep now :)

**Review by ekkinox — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (Skyld) [reply composer.json]: So it cannot be merged I guess?

**Review by Skyld — CHANGES_REQUESTED** [review]

**Comment by julien-sebire** [comment]
@Skyld It won't be merged to develop but to a feature branch to gather all the modifications to be deployed to Google Platform for the benchmark. It will only be merged to develop whenever we reach the end of the benchmark and first version of lib-dbal-spanner.

**Review by Skyld — APPROVED** [review]

**Review by Skyld — CHANGES_REQUESTED** [review]
Please fix conflicts

**Comment by jbout** [comment]
update already happened

**Pull request closed without merging**